### PR TITLE
Fix expired token

### LINKED ISSUE #526: Fix Expired Token Error with Host Toggle
# Background

- The host toggle only works once before an expired token error pops up.
# Task

- Find and fix the bug.

# Done When

- Can switch the host toggle unlimited amount of times

### PR DESCRIPTION
**Pull request by syd-tan**
# Fix Expired Token Error on Host Toggle

Closes #526

# Summary of change

- Previously, the token in the Login redux state would disappear after calling setUserInfo route once.
- Fixed bug by passing in token from Login rather than User.

# Testing/Verification

- Should now be able to click on the switch to host/non-host toggle in the settings tab under My Account unlimited amount of times.

### CODE REVIEW

File: src/components/myAccount/settings.component.js [hunk 1]
```diff
@@ -4,31 +4,41 @@ import { connect } from "react-redux";
...
     this.state = {
       localIsHost: props.User.userInfo.isHost,
+      token: props.Login.userInfo.session.token,
     };
   }
 
   changeHostStatus = async () => {
     this.state.localIsHost = !this.state.localIsHost;
     const userId = this.props.User.userInfo._id;
+    console.log(this.state.token);
```
Comment (aidenszeto) [inline src/components/myAccount/settings.component.js:21]: Could we remove this console log?

**Review by nishantbalaji — APPROVED** [review]
works

**Comment by syd-tan** [comment]
@aidenszeto removed console.log

**Review by aidenszeto — APPROVED** [review]
LGTM! Could you remove the package-lock changes?

**Comment by nishantbalaji** [comment]
> LGTM! Could you remove the package-lock changes?

^ These should be ignored and the file deleted from the PR so it doesn't get merged

**Pull request merged**